06_gpu_and_ml/stable_lm/main.py

```diff
@@ -0,0 +1,146 @@
+import time
+from pathlib import Path
+
+import torch
+from huggingface_hub import snapshot_download
+from pydantic import BaseModel, Field
+from rich import print
+from transformers import (
+    AutoModelForCausalLM,
+    AutoTokenizer,
+    GenerationConfig,
+    StoppingCriteria,
+    StoppingCriteriaList,
+    pipeline,
+)
```

Comment:
```suggestion
from transformers import (
    StoppingCriteria,
    StoppingCriteriaList,
    pipeline,
)
```

Ideally we don't need to import `transformers` in global scope, because it isn't used locally.